itest/godot/project.godot

```diff
@@ -13,3 +13,8 @@ config_version=5
 config/name="IntegrationTests"
 run/main_scene="res://TestRunner.tscn"
 config/features=PackedStringArray("4.0")
+run/flush_stdout_on_print=true
+
+[debug]
+
+gdscript/warnings/shadowed_variable=0
```

Comment:
Why this? Also, should we disable that warning globally?

Reply:
Because it's overzealous. I'm shadowing a field with a constructor arg in several places, e.g. `self.suite = suite`. Want me to use something like `suite_` for such args instead?

Reply:
Ah no, then it makes sense, thanks for elaborating 👍